meta-balena-reference/layers/meta-balena-bsp/recipes-bsp/grub/files/cfg

```diff
@@ -0,0 +1,2 @@
+set root=hd0,msdos1
```

Comment:
Is grub so different for Qemu than for x86 or x86-64? It may be they are really similar and the common changes belong in meta-balena-common.

Reply:
@alexgg grub.cfg needs to be harmonised and pulled into meta-balena..

https://github.com/balena-os/meta-balena/issues/1728